### DISCUSSION TITLE
🤖 Get your Deepgram questions answered instantly by our AI Bot!

### DISCUSSION THREAD

**Discussion by jpvajda** [deepgram/community#862] [News + Announcements]
Don't want to wait for an answer to your Community question? 

Try out our AI Bot on [Deepgram Docs](https://developers.deepgram.com/docs/introduction) which can help you get your questions answered faster!

![Screenshot 2024-07-19 at 2 24 21 PM](https://github.com/user-attachments/assets/a2c7c6ff-62e0-42c2-87f9-ca053a860367)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
